.github/ISSUE_TEMPLATE/bugreport.yml

```diff
@@ -0,0 +1,77 @@
+name: Bug Report
+description: You found a bug on Scuffle? This way please
+title: "[Bug]: Scuffle has a bug here [...]"
+labels: ["bug"]
+assignees:
+  - 
```

Comment:
Who exactly should we assign to the issues?

Reply:
@TroyKomodo

Reply:
How do u mean assign?

Reply:
like who does the work?

Reply:
We can use it to automatically assign people in case of e.g. bugs or feature requests.

For example, if you are the contact person for bugs, you will be added automatically.

Reply:
hmm, perhaps the reviewer team?

Reply:
Need to test if Github supports this by now.